src/ape/types/__init__.py

```diff
@@ -75,11 +80,20 @@ def __getattr__(self, item: str) -> Any:
             item (str): The name of the property.
         """
 
+        try:
```

Comment:
TBD - do we just rid of `__getattr__` due to potential naming conflicts + the myriad of issues we see when using `__getattr__` and serialization?

- [ ] Decide and / or take action

Reply:
i think replacing it with `__getitem__` would be fine

Reply:
In general, our guideline should be to never use `__getattr__` internally when other methods would do.

It's a great UX feature to be able to use it, but it should be possible to reference via other means as well in order to prevent infinite loops and other hard to debug situations

Reply:
OK - going to leave it and just not call it anywhere! 🤝